zebedee-cms/src/main/java/com/github/onsdigital/zebedee/service/StaticFilesServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.github.onsdigital.zebedee.service;
+
+import com.github.onsdigital.dp.files.api.Client;
+import com.github.onsdigital.zebedee.model.Collection;
+
+
+public class StaticFilesServiceImpl implements StaticFilesService {
+
+
+    private final Client client;
+
+    public StaticFilesServiceImpl(Client client) {
+        this.client = client;
+    }
+
+    @Override
+    public void publishCollection(Collection collection){
+        if (collection.getDescription() == null) {
+            throw new IllegalArgumentException("Collection description was null.");
```

Comment:
I don't know Java that well but shouldn't the description of the IllegalArgumentException be in lower case?